guess.js

```diff
@@ -147,14 +156,22 @@ function guessCheck() {
         // span.append(text)
         // massageArea.append(text2, span)
         //// OR esily///
+        // [REVIEW] Usually I suggest creating elements instead of directly setting the innerHTML.
+        // .. this is because it's easier to manage the elements and their attributes later.
+        // [REVIEW] It would be nice if you can create a little function to update the message area text ..
+        // since you are repeating the same code in multiple places.

```

Comment:
like a onr way  ?

Reply:
Yes, like the above [implementation](https://github.com/mohamed-sabry-1789/guess-word-game/blob/main/guess.js#L144-L148).

Reply:
why the second why easir